contracts/mainnet/connectors/uniswapV3/main.sol

```diff
@@ -0,0 +1,145 @@
+pragma solidity =0.7.6;
+pragma abicoder v2;
+
+/**
+ * @title Uniswap v3.
+ * @dev Decentralized Exchange.
+ */
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {Helpers} from "./helpers.sol";
+import {Events} from "./events.sol";
+
+abstract contract UniswapResolver is Helpers, Events {
+    /**
+     * @dev Mint New Position
+     * @param params: parameter for mint.
+     * @param getId: ID to retrieve amtA.
+     * @param setId: ID stores the amount of LP token.
+     */
+    function mint(
+        MintParams memory params,
+        uint256 getId,
+        uint256 setId
+    )
+        external
+        payable
+        returns (string memory _eventName, bytes memory _eventParam)
+    {
+        uint256 _amt = getUint(getId, params.amtA);
+        params.amtA = _amt;
+        (
+            uint256 _tokenID,
+            uint256 _amtA,
+            uint256 _amtB,
+            uint256 liquidity
+        ) = _mint(params);
+        setUint(setId, liquidity);
+
+        _eventName = "LogMintNewPosition(uint256,uint256,uint256,uint256)";
+        _eventParam = abi.encode(_tokenID, _amtA, _amtB, liquidity);
+    }
+
+    /**
+     * @dev Increase Liquidity
+     * @param tokenId: NFT LP Token ID.
+     * @param amountA: tokenA amounts.
+     * @param amountB: tokenB amounts.
+     * @param getIds: IDs to retrieve token amounts
+     * @param  setId: stores the amount of LP token
+     */
+    function addLiquidity(
```

Comment:
@cryptoDev222 how are we sending amount0Min and amount1Min?

Reply:
@cryptoDev222 same for decreaseLiquidity

Reply:
That's just 0.

Reply:
Okay, I'll fix it.